lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -63,7 +63,6 @@
 
   private static final String RUNTIME_NAME = "YARN";
 
-  private final YarnClient yarnClient = YarnClient.createYarnClient();
```

Comment:
do we still need
`import org.apache.hadoop.yarn.client.api.YarnClient;`
after this change?